src/renderer/component/publishForm/view.jsx

```diff
@@ -442,6 +444,22 @@ class PublishForm extends React.PureComponent {
     this.setState({ id });
   }
 
+  componentWillReceiveProps(nextProps) {
+    if (nextProps.uploadStatus === "error") {
+      this.props.alertError("Upload failed. Please try again.");
+      this.setState({
+        thumbnailUploadStatus: "upload",
+      });
+    }
+
+    if (nextProps.uploadStatus === "complete") {
+      this.setState({
+        thumbnailUploadStatus: "complete",
+        meta_thumbnail: nextProps.uploadUrl,
```

Comment:
where do you use thumbnailUploadStatus and meta_thumbnail? I don't see why this is necessary